src/odemis/acq/stream/_base.py

```diff
@@ -523,9 +553,18 @@ def _setStatus(self, level, message=None):
         self.status.notify(self.status.value)
 
     def onTint(self, value):
-        if self.raw:
-            data = self.raw[0]
-            data.metadata[model.MD_USER_TINT] = value
+        if isinstance(self.raw, list):
+            if len(self.raw) > 0:
+                raw = self.raw[0]
+            else:
+                raw = None
+        elif isinstance(self.raw, tuple):
+            raw = self._das
```

Comment:
Here is a bit tricky. In the case of static streams, updating the metadata only matters when saving a "snapshot". This feature uses the .raw tiles (and I think indirectly the metadata of the first tile, as is done in mergeTile()).
So to do so properly, you'd need to both update the first DA metadata (if it's immediatly saved) and the DAS metadata (in case the tiles are updated before saving). In practice, it'll not matter very much. So leave it as-is for now, but add a comment explaining this metadata will trickle-down to the raw tiles as soon as they are updated.